robolectric/src/test/java/org/robolectric/shadows/ShadowPendingIntentTest.java

```diff
@@ -25,63 +33,54 @@ public void getBroadcast_shouldCreateIntentForBroadcast() throws Exception {
     assertThat(shadow.isActivityIntent()).isFalse();
     assertThat(shadow.isBroadcastIntent()).isTrue();
     assertThat(shadow.isServiceIntent()).isFalse();
-    assertThat(intent).isEqualTo(shadow.getSavedIntent());
     assertThat((Context) RuntimeEnvironment.application).isEqualTo(shadow.getSavedContext());
     assertThat(shadow.getRequestCode()).isEqualTo(99);
     assertThat(shadow.getFlags()).isEqualTo(100);
   }
 
   @Test
-  public void getActivity_shouldCreateIntentForBroadcast() throws Exception {
+  public void getActivity_shouldCreateIntentForActivity() throws Exception {
     Intent intent = new Intent();
     PendingIntent pendingIntent = PendingIntent.getActivity(RuntimeEnvironment.application, 99, intent, 100);
     ShadowPendingIntent shadow = shadowOf(pendingIntent);
     assertThat(shadow.isActivityIntent()).isTrue();
     assertThat(shadow.isBroadcastIntent()).isFalse();
     assertThat(shadow.isServiceIntent()).isFalse();
-    assertThat(intent).isEqualTo(shadow.getSavedIntent());
```

Comment:
Instead of removing these assertion, perhaps add an action to the intent and compare the action?